### ISSUE TITLE
Listing: Facility name workaround for H14

### ISSUE THREAD

**Issue by ifoche** [EyeSeeTea/QAApp#13]
Feedback from gtsmith@psi.org : The health facility name should take its own row if possible. Everything beneath it would align a little cleaner if we do that.  Not sure if that is possible though.  It is a nice to have

 [Link to buddybuild feedback from build 4](https://dashboard.buddybuild.com/apps/56b408ad65c5670100adf4df/feedback?fid=56d08e392447a70100658c26&bnum=4)

<img src="https://s3-us-west-2.amazonaws.com/buddybuild-screenshots/56b408ad65c5670100adf4df/56d057fac8f4aa010054eac5/938ee9b5-abba-440c-a888-eec6c4626742.jpg" width="33%" height="33%" /><table><tr><td>Build</td><td>4</td></tr><tr><td>Device type</td><td>SM-T217S</td></tr><tr><td>Screen size</td><td>600</td></tr><tr><td>Screen size</td><td>600px by 1024px</td></tr><tr><td>Battery</td><td>40% Unplugged</td></tr><tr><td>Memory free</td><td>182 MB / 1347 MB</td></tr></table>

**Comment by ifoche** [comment]
@gtrsmith I talked about this with Rodolfo, to accomplish that, it would be necessary a refactor of a part of the code that would imply a little bit more development than it probably may appear. We can do it if you find it necessary, but I wanted just to prevent you and get your confirmation before starting that.

**Comment by gtrsmith** [comment]
Rodolfo and Ignacio should discuss how complex this is in order to determine if this should be H14 or H15

**Comment by rodmelia** [comment]
Given the complexity involved this will be for H15.
We should also include this issue on a overall review of the UX.

**Comment by ifoche** [comment]
We've found a workaround and included it in the last build (but still the code management is needed to solve th e bug that appears when you slide down and up again in that assess list)

**Comment by rodmelia** [comment]
I;ve added a card for H15, so we can spend more time on this. 
@ifoche :  how does the current work around looks like? Can you attach a screenshot?

**Comment by ifoche** [comment]
sure! 

![one_line_orgunit_name](https://cloud.githubusercontent.com/assets/10931645/13950045/f9b3cadc-f028-11e5-8946-c864d6926677.png)

**Comment by rodmelia** [comment]
This looks great !

**Comment by cristinaelle** [comment]
@ifoche: nice! well done!

**Comment by gtrsmith** [comment]
closing this ticket - looks good
